src/patternfly/components/LogViewer/examples/LogViewer.md

```diff
@@ -120,6 +121,7 @@ import './LogViewer.css';
 | `.pf-c-log-viewer__timestamp` | `<div>` | Initiates a log viewer text element. **Required** |
 | `.pf-m-wrap-text` | `.pf-c-log-viewer` | Modifies the log viewer text to wrap. |
 | `.pf-m-line-numbers` | `.pf-c-log-viewer` | Modifies the log viewer to display line numbers. |
+| `.pf-m-dark` | `.pf-c-log-viewer` | Modifies the log viewer for dark theme. |
```

Comment:
My only comment is maybe here say that it "modifies the log viewer *content*" so that users know that it's not modifying the toolbar at this time (if ever?)

Reply:
@srambach updated